scripts/docker/dockerfile_clang-4

```diff
@@ -0,0 +1,30 @@
+FROM axom/compilers:clang-4
+
+RUN sudo apt-get update -y
+RUN sudo apt-get install curl -fy
+
+RUN sudo apt-get -qq install -y --no-install-recommends gfortran-7 && sudo update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-7 100
+ 
+#(For Clang Builds - Need to restore stdc++ lib)
+RUN sudo apt-get install libstdc++-7-dev
+
+RUN git clone --recursive https://github.com/LLNL/axom.git
+
+RUN git clone --recursive https://github.com/LLNL/RAJA.git
+
+WORKDIR "/home/axom/RAJA"
+RUN mkdir build
+
+WORKDIR "/home/axom/RAJA/build"
+
+RUN cmake ../. -DENABLE_TESTS=OFF
+RUN make 
+RUN sudo make install
+
```

Comment:
This probably doesn't matter since we're going to fix the RAJA build problem, 
but should we delete the ``raja/build`` directory?